ansible_rulebook/event_filter/noop.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any
```

Comment:
@andresberejnoi This might not be needed unless there is a test pointing to this.